openlibrary/plugins/openlibrary/js/index.js

```diff
@@ -564,3 +564,16 @@ jQuery(function () {
             .then(module => module.initAffiliateLinks(affiliateLinksSection))
     }
 });
+
+// Dark Mode toggle
+const darkMode = document.getElementById('darkMode')
+darkMode.addEventListener('click', function() {
```

Comment:
`darkMode` may be `null` at this point (it only exists on the settings page).  It would be better to pass `darkMode` as a parameter of `initDarkMode`, and add the event listener in the body of `initDarkMode`.

Reply:
thanks for highlighting this, I can see that this is how the other .js in structured in index.js. I'll submit the updated PR shortly